packages/docs/src/root.tsx

```diff
@@ -16,6 +16,21 @@ export default component$(() => {
 
   useContextProvider(GlobalStore, store);
 
+  useVisibleTask$(({ cleanup }) => {
```

Comment:
Could we solve this without `useVisbelTask$` which forces code to run eagerly?

Reply:
i'll look into it

Reply:
I think you want `useOnDocument('click', $(...));` https://qwik.builder.io/tutorial/hooks/use-on/